open_spiel/algorithms/mcts.cc

```diff
@@ -200,11 +200,12 @@ std::vector<double> dirichlet_noise(int count, double alpha,
   return noise;
 }
 
-MCTSBot::MCTSBot(const Game& game, std::shared_ptr<Evaluator> evaluator,
+MCTSBot::MCTSBot(const Game &game, std::shared_ptr<Evaluator> evaluator,
```

Comment:
Revert whitespace change (these sorts of changes just add noise to the diff)

Reply:
Changed